src/Native/Runtime/arm64/PInvoke.S

```diff
@@ -287,8 +292,13 @@ NESTED_ENTRY RhpPInvoke, _TEXT, NoHandler
         str     x9, [x0, #OFFSETOF__PInvokeTransitionFrame__m_Flags]
 
         // get TLS global variable address
-        // r0 = GetThread()
+
+#ifdef FEATURE_EMULATED_TLS
+        GETTHREAD_ETLS_10
```

Comment:
This does not need to preserve all x0-x9

Reply:
OK, I wrongly assumed that the need to be preserved as the code was not touching them. I changed x10 to x1 to void adding another of these macro variants.